### PR TITLE
Fix unmanaged branch notification not showing up

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

File: branchLayout/api/src/main/java/com/virtuslab/branchlayout/api/BranchLayoutException.java [hunk 1]
```diff
@@ -24,13 +26,4 @@ public BranchLayoutException(@Nullable @Positive Integer errorLine, String messa
   public BranchLayoutException(String message, Throwable e) {
     this(null, message, e);
   }
-
-  public @Nullable @Positive Integer getErrorLine() {
-    return errorLine;
-  }
-
-  @Override
-  public String toString() {
```
Comment (PawelLipski) [inline branchLayout/api/src/main/java/com/virtuslab/branchlayout/api/BranchLayoutException.java:33]: ...plus some unrelated minor fixes

File: frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java [hunk 2]
```diff
@@ -295,7 +295,7 @@ private void trackCurrentBranchChange(GitRepository repository) {
           unmanagedBranchNotification.expire();
         }
         val snapshot = gitMacheteRepositorySnapshot;
-        val mainGitDirectory = GitVfsUtils.getMainGitDirectory(repository);
+        Path mainGitDirectory = GitVfsUtils.getMainGitDirectory(repository).toNioPath();
```
Comment (mkondratek) [inline frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java:298]: 👍 👍 👍

**Review by mkondratek — APPROVED** [review]
